images/ubuntu2004/scripts/cloudinit_install.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/sh
+
+GRUB_CMDLINE_LINUX_DEFAULT="console=tty1 console=ttyS0,115200n8 earlyprintk=ttyS0,115200 rootdelay=300 quiet splash"
+
+echo "ubuntu" | sudo -S apt update
```

Comment:
We should run the entire script with sudo by setting the `execute_command` on the provisioner in the packer config

https://www.packer.io/docs/provisioners/shell#sudo-example

Reply:
Thanks, updated